src/extensions/step_handlers/apt_handler/tests/CMakeLists.txt

```diff
@@ -23,8 +23,8 @@ target_include_directories (
             ${PROJECT_SOURCE_DIR}/inc ${PROJECT_SOURCE_DIR}/../../inc ${PROJECT_SOURCE_DIR}/../inc)
 
 target_compile_definitions (
-    ${PROJECT_NAME} PRIVATE ADUC_INSTALLEDCRITERIA_FILE_PATH="/tmp/adu-test-installedcriteria"
-                            ADUSHELL_FILE_PATH="${ADUSHELL_FILE_PATH}")
+    ${PROJECT_NAME}
+    PRIVATE ADUC_INSTALLEDCRITERIA_FILE_PATH="/${ADUC_TMP_DIR_PATH}/adu-test-installedcriteria")
```

Comment:
ADUSHELL_FILE_PATH is removed. Intentional?

Reply:
weird. let me go check